src/javascripts/ng-admin/Crud/delete/DeleteController.js

```diff
@@ -22,14 +22,11 @@ define(function () {
 
     DeleteController.prototype.deleteOne = function () {
         var notification = this.notification,
-            $state = this.$state, entityName = this.entity.name();
+            entityName = this.entity.name(),
+            $window = this.$window;
 
         this.WriteQueries.deleteOne(this.view, this.entityId).then(function () {
-
-            $state.go($state.get('list'), angular.extend({
-                entity: entityName,
-                id: this.entityId
-            }, $state.params));
+            this.back();
```

Comment:
@ahgittin this is taken care of in #642. Could you test it and report your findings in the PR?